pkgs/data/themes/sweet/default.nix

```diff
@@ -1,29 +1,62 @@
-{ lib, stdenv, fetchurl, unzip, gtk-engine-murrine }:
+{ lib
+, stdenvNoCC
+, fetchurl
+, unzip
+, gtk-engine-murrine
+}:
 
-stdenv.mkDerivation rec {
+stdenvNoCC.mkDerivation rec {
```

Comment:
Since this PR already has some cleanup, would you mind editing it to use `finalAttrs` as well?

No big deal if you feel like it's enough cleanup as it is.

Reply:
@h7x4 sure. I fixed it.